src/content/security/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Ethereum security and scams
+description: Staying safe on Ethereum
+lang: en
+sidebar: true
+---
+
+## Staying Safe on Ethereum
+
+With interest in cryptocurrencies growing exponentially, learning best practices when using cryptocurrency is essential. Crypto can be fun and exciting, but there are also serious risks. If you put in this small amount of upfront work, you can mitigate these risks.
+
+<Divider />
+
+## Web Security 101
+
+### Use strong passwords
+
+Over 80% of account hacks are a result of weak or stolen passwords. A long combination of characters, numbers and symbols is best to keep your accounts secure.
+
+A common mistake individuals make is using a long combination of common dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](<[https://en.wikipedia.org/wiki/Dictionary_attack](https://en.wikipedia.org/wiki/Dictionary_attack)>).
```

Comment:
Yeah I kinda missed the point I was making. Fixed to say this:

> A common mistake individuals make is using a combination of two to three common, related dictionary words. Passwords like this are insecure because they are prone to a simple hacking technique known as a [dictionary attack](<[https://en.wikipedia.org/wiki/Dictionary_attack](https://en.wikipedia.org/wiki/Dictionary_attack)>).

I think all of the information you suggested is probably suitable for an advanced section. Happy to iterate on this with you because you're definitely more knowledgeable.